web/js/components/Menu.jsx

```diff
@@ -105,10 +105,10 @@ export class WaitingRoom extends React.Component {
             </div>
             {this.props.isHost ? 
               <div>
-                <button className="action-button top-button" onClick={this.toggleModal}>
+                <button className="action-button top-button fixed-width" onClick={this.toggleModal}>
```

Comment:
where is fixed-width defined?

I think this class isn't a match for anything.

Reply:
forgot to hit save on my css file 😇 - fixed